jenkins/run-server-func-tests

```diff
@@ -76,7 +77,7 @@ function dump_journal {
 }
 
 function cleanup {
-    if (( ${cleanup_flag} )); then
+    if (( cleanup_flag )); then
```

Comment:
On a whim, I installed the `shellcheck` extension in `vscode`, partly to identify exactly what you were overriding with your disable directives ... but I see this whole block marked `SC2317` because apparently the linter doesn't understand how `cleanup` gets called through the `trap`. Are you not seeing that, or did you just choose to ignore it?

Reply:
I looked up `SC2317`, and I agree that it should, _erroneously_, be flagged here by ShellCheck (the [description](https://www.shellcheck.net/wiki/SC2317#exceptions) explicitly says that, because it's invoked via a `trap`, we should ignore its warning), but my IDE doesn't flag it.  I don't know whether I've disabled/ignored the warning or whether you and I are simply running different versions of SheckCheck and mine is smarter/dumber than yours.  🙃